src/Core/Notifications/NotificationAction.cs

```diff
@@ -32,8 +32,8 @@ public class NotificationAction : INotificationAction
     {
         public NotificationAction(string commandText, Action action)
```

Comment:
Just some sanity checks, encountered this during my tests for the notification service by mistake.